raspbian/applications/vcgencmd.md

```diff
@@ -15,7 +15,7 @@ Some of the more useful  commands are described below.
 
 #### vcos
 
-The `vcos` cammand has a number of sub commands
+The `vcos` cammand has a number of sub commands.
 
```

Comment:
Cammand?  yes, I know that's what the original said as well!!!

Reply:
That's the difference between one of my PRs and one of @lurch's - my eyes see only some of the mistakes 😁